std/experimental/color/package.d

```diff
@@ -0,0 +1,500 @@
+// Written in the D programming language.
+
+/**
+    This package contains implementations of various common _color types.
+    Types are supplied for various _color spaces, along with _color space
+    conversion functionality.
+
```

Comment:
How about adding a last sentence with

_Each function and type defined in the package is completely safe, nothrow, nogc and pure_.  ?

The problem is since everything is templatized it doesn't even appear on the doc.